dissect/target/loader.py

```diff
@@ -194,6 +194,7 @@ def open(item: Union[str, Path], *args, **kwargs):
 register("tanium", "TaniumLoader")
 register("itunes", "ITunesLoader")
 register("target", "TargetLoader")
+register("log", "LogLoader")
```

Comment:
EvtLogLoader only covers the current functionality (evt + evtx), this loader can easily be extended to parse other logs as well (systemd journal files, etl etc). I first named this loader "FileLoader" but after careful inspection I decided I could narrow it down to LogLoader because logs are most likely to exist "outside" a target (registry hives are often integrated with the target, so they are less likely to exist on their own).